src/components/layout/CompanyClaim/CompanyClaim.js

```diff
@@ -30,6 +30,14 @@ const CompanyClaim = () => (
           </a>
         </div>
       </div>
+      <div className={styles.bannerMobileRow}>
```

Comment:
Jeśli da sie na CSS, to lepiej ogarnąć to na CSS niż robić drugi HTML z tym samym elementem ;) Możesz ustawić elementy za pomocą `order`, następnie dla numeru telefonu ustawić `flex-basic` na 100% co spowoduje że spadnie poniżej loga i koszyka, teraz już zostaje wyrównać tekst do prawej ;)

Reply:
Dzieki, tego mi zabrakło. Wygląda na to że działa. Wypchnąłem commit, czy mam zrobić nowy pull request teraz?